client/modules/IDE/components/Editor/index.jsx

```diff
@@ -197,6 +201,16 @@ class Editor extends React.Component {
     });
 
     this._cm.on('keydown', (_cm, e) => {
+      if (
+        ((metaKey === 'Cmd' && e.metaKey) ||
+          (metaKey === 'Ctrl' && e.ctrlKey)) &&
+        e.shiftKey &&
+        e.key === 'f'
+      ) {
+        e.preventDefault();
+        this.tidyCode();
+      }
+
```

Comment:
The removal on line 164 and the addition here is due to a change made to the develop branch in another PR.  Yours should be the same as the current.